dan_layer/engine/tests/templates/nft/emoji_id/src/lib.rs

```diff
@@ -65,35 +85,37 @@ mod emoji_id {
             }
         }
 
-        // TODO: how do we ensure that the payment is indeed in Thaums?
-        pub fn mint(&mut self, emojis: Vec<Emoji>, payment: Bucket) -> (Bucket, Bucket<Thaum>) {
+        // TODO: return change
```

Comment:
Maybe instead of returning change we just check the value `bucket.value() == required_amount`